core/src/main/java/org/apache/calcite/sql/dialect/SparkSqlDialect.java

```diff
@@ -206,6 +206,7 @@ public class SparkSqlDialect extends SqlDialect {
   private static final String AND = "&";
   private static final String OR = "|";
   private static final String XOR = "^";
+  private static final String RLIKE = "rlike";
```

Comment:
is this required? if we are using writer can simply not simply use writer.print("RLIKE")